### DISCUSSION TITLE
need info on the CSI_PLUGIN_PRIORITY_CLASSNAME and CSI_PROVISIONER_PRIORITY_CLASSNAME

### DISCUSSION THREAD

**Discussion by RajuMedpur** [rook/rook#9787] [General]
As per the templates provided for rook-ceph-operator.yaml , if pod priority is enabled below are the values. 
CSI_PLUGIN_PRIORITY_CLASSNAME: "system-node-critical"
CSI_PROVISIONER_PRIORITY_CLASSNAME: "system-cluster-critical" 

are we need to use system defined priority classes for the above ? 
can we use my-priority-class classes defined for the application ? please confirm

**Comment by travisn** [comment]
@RajuMedpur This can be a custom priority class, those values are just examples. Whatever the value for those settings, rook will just set them as the `priorityClassName` property on the pod specs.

**Reply by RajuMedpur** [reply]
Thanks for the quick reponse @travisn
